.gitignore

```diff
@@ -6,3 +6,7 @@ tmp/
 .nyc_output/
 package-lock.json
 yarn.lock
+*_BACKUP_*
+*_BASE_*
+*_LOCAL_*
+*_REMOTE_*
```

Comment:
?

Reply:
Those files are generated by Git for Windows in case of conflict. It is common to exclude them, it was not done, so I did it.

Reply:
dropped it